### PR TITLE
remove redundant methods invoke from SubtitleListView

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Forms/DurationsBridgeGaps.cs [hunk 1]
```diff
@@ -126,6 +124,7 @@ private void GeneratePreviewReal()
             FixedCount = Core.Forms.DurationsBridgeGaps.BridgeGaps(_fixedSubtitle, minMsBetweenLines, radioButtonDivideEven.Checked, (double)numericUpDownMaxMs.Value, fixedIndexes, _dic);
 
             SubtitleListview1.Fill(_fixedSubtitle);
```
Comment (ivandrofly) [inline src/Forms/DurationsBridgeGaps.cs]: `Fill` calles Items.Clear() + BeginUpdate() / EndUpdate()

File: src/Forms/DurationsBridgeGaps.cs [hunk 2]
```diff
@@ -145,7 +144,6 @@ private void GeneratePreviewReal()
             {
                 SubtitleListview1.SetBackgroundColor(index, Color.LightGreen);
             }
-
             SubtitleListview1.EndUpdate();
```
Comment (niksedk) [inline src/Forms/DurationsBridgeGaps.cs]: You probably should remove "SubtitleListview1.EndUpdate();" also

File: src/Forms/DurationsBridgeGaps.cs (hunk 1 above)
Comment (niksedk) [reply src/Forms/DurationsBridgeGaps.cs]: I think this is okay, it clears the list view, and handles a lot of listview stuff (BeginUpdate can be nested I think)

Comment (ivandrofly) [reply src/Forms/DurationsBridgeGaps.cs]: > I think this is okay, it clears the list view, and handles a lot of listview stuff (BeginUpdate can be nested I think)

Sure, that is why we don't need remove lines above! I don't know if you noticed, I didn't remove BeginUpdate / EndUpdate.

**Pull request closed without merging**